README.rst

```diff
@@ -75,9 +75,46 @@ From System Package Manager
 ---------------------------
 
 If your operating system packages repolib, you can install it by running::
-    
+
     sudo apt install python3-repolib
 
+If your distro does not yet supply the package, you can install the
```

Comment:
I think the new section should have its own title, maybe `From Pre-Compiled .deb Package`. If you want to change the above `From System Package Manager` to `From System Repository` to make the difference more clear, you could do that.